live-examples/css-examples/basic-user-interface/outline-offset.css

```diff
@@ -0,0 +1,7 @@
+#example-element {
+    background-color: #ffd955;
+    outline: .75em solid;
+    padding: .75em;
```

Comment:
perhaps also add a border around it to demonstrate the offset between the outline and the border?